test/api._coffee

```diff
@@ -120,7 +133,7 @@ expectDZI = (act, exp={}) ->
     for prop in ['width', 'height', 'tileSize', 'tileOverlap']
         _expectNumberWithin act[prop], 0, Infinity
 
-    _expectStringMatching act.tileFormat, /^(png|jpeg)$/
+    _expectStringMatching act.tileFormat, /^(png|jpg|jpeg)$/
```

Comment:
I might be getting mixed up with our internal Mix API at @FiftyThree though. We use the `image/` MIME subtype, which is always `jpeg` and never `jpg`.